powersimdata/design/investment/investment_costs.py

```diff
@@ -59,27 +84,35 @@ def select_mw(x, cost_df):
         :return: (*pandas.Series*) -- series of [*'MW'*, *'costMWmi'*] to be assigned
             to branch.
         """
-
+        underground_regions = ("NEISO", "NYISO J-K")
+        cost_df = cost_df.copy()
+        # Unless we are entirely within an underground region, drop this cost class
+        if not (x.from_region == x.to_region and x.from_region in underground_regions):
+            cost_df = cost_df.query("kV != 345 or MW != 500")
         # select corresponding cost table of selected kV
-        tmp = cost_df[cost_df["kV"] == x.kV]
+        cost_df = cost_df[cost_df["kV"] == x.kV]
         # get rid of NaN values in this kV table
-        tmp = tmp[~tmp["MW"].isna()]
+        cost_df = cost_df[~cost_df["MW"].isna()]
         # find closest MW & corresponding cost
-        return tmp.iloc[np.argmin(np.abs(tmp["MW"] - x.rateA))][["MW", "costMWmi"]]
+        return_cols = ["MW", "costMWmi"]
+        return cost_df.iloc[np.argmin(np.abs(cost_df["MW"] - x.rateA))][return_cols]
 
-    def get_transformer_mult(x, bus_reg, ac_reg_mult, xfmr_lookup_alerted=set()):
+    def get_branch_mult(x, bus_reg, ac_reg_mult, branch_lookup_alerted=set()):
```

Comment:
I think based on our convention, we should avoid mutable default values?

Reply:
This design is intentional. Since we re-define `get_branch_mult` for every call of `_calculate_ac_inv_costs`, it is never shared via user-accessible calls. We do want it to be shared via the many calls to `get_branch_mult` via the dataframe `apply`, so that it serves its purpose of not the user multiple times for the same missing regional multipliers.

I believe this was discussed at some point in the review for https://github.com/Breakthrough-Energy/PowerSimData/pull/297.

Reply:
Nevermind then.